docs/README-develop-node-utils-APIs.md

```diff
@@ -159,6 +159,7 @@ The following basic command_reference parameters will be defined for each resour
  4. `default_value:` This is typically the "factory" default state of the property, expressed as an actual value (true, 12, "off", etc)
  5. `kind:` The data type of this property. If omitted, the property will be a string by default. Commonly used values for this property are `int` and `boolean`.
  6. `multiple:` By default a property is assumed to be found once or not at all by the `get_command`/`get_value` lookup, and an error will be raised if multiple matches are found. If multiple matches are valid and expected, you must set `multiple: true` for this property.
+ 7. `os_version:` By default a property is assumed to be supported on all version of software. If it is supported starting from a particular version, then this field can be used to specify the version for a particular product_id.
```

Comment:
Also please indicate that this is a `test_only` feature and is ignored outside the context of the minitests.